views/scoreCriteriaTemplate.ejs

```diff
@@ -0,0 +1,54 @@
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+
+    <%if(data.schoolName) {%>
```

Comment:
@deepakr123 - Rename it to the entity name

Reply:
done